### ISSUE TITLE
[docs][bug] Accessibility > Design page missing images

### ISSUE THREAD

**Issue by hellogreg** [RedHat-UX/red-hat-design-system#2183]
Looks like we've got a couple broken images on the [Accessibility > Design](https://ux.redhat.com/accessibility/design/) page, in the Color and Layering sections.

Corey also recommends restoring borders to the images.
